benchmark/utils/general_matrix.hpp

```diff
@@ -41,12 +41,84 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 #include "benchmark/utils/general.hpp"
+#include "benchmark/utils/generator.hpp"
+
+
+std::string reordering_algorithm_desc =
+    "Reordering algorithm to apply to the input matrices:\n"
+    "    none - no reordering\n"
```

Comment:
maybe 
```suggestion
    "    none - no reordering (default)\n"
```

Reply:
The default value is already displayed in the `--help` output